src/utils/isSupportedFileExtension.ts

```diff
@@ -0,0 +1,11 @@
+export function isSupportedFileExtension(path: string | undefined) {
+  const validExtensions = ['.js', '.ts', '.tsx'];
+
+  if (path === undefined) {
+    return false;
+  }
```

Comment:
Maybe let's return early if no `path` available.
 
```suggestion
  if (path === undefined) {
    return false;
  }
  
  const validExtensions = ['.js', '.ts', '.tsx'];
```

Reply:
@brunokiafuka I have added changes based on your feedback.